app/views/admin/categories/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<% @categories.each do |cat| %>
+  <%= cat.title %>
+  <%= cat.id %>
+<% end unless @categories.blank? %>
```

Comment:
it's better to avoid using `unless` with multiline expressions, I think in the current case there is no need for such condition at all

Reply:
it gives error if @categories are empty